packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu.ts

```diff
@@ -134,11 +134,17 @@ export class PharosDropdownMenu extends FocusMixin(OverlayElement) {
         (this._currentTrigger as PharosDropdownMenuNavLink).isActive = this.open;
       }
 
+      if (this.open && !this._popper) {
+        this._setupMenu();
+      }
+
       if (
         this.open &&
-        (!this._popper || this._popper?.state.elements.reference !== this._currentTrigger)
+        this._popper &&
+        this._popper.state.elements.reference !== this._currentTrigger
```

Comment:
Can this condition be named? Getting difficult to tell exactly what this represents.